astrocut/tests/test_make_cube.py

```diff
@@ -48,6 +49,56 @@ def test_make_cube(tmpdir):
     hdu.close()
     
 
+def test_update_cube(tmpdir):
```

Comment:
can you rename this?

```suggestion
def test_make_and_update_cube(tmpdir):
```